api/v1/tetragon/tetragon.proto

```diff
@@ -170,6 +176,7 @@ message KprobeArgument {
 	KprobeSock sock_arg = 9;
 	KprobeCred cred_arg = 10;
 	int64 long_arg = 11;
+    KprobeBpfAttr bpf_attr_arg = 12;
```

Comment:
Please fix also this formatting issue. I think that our linters do not run on the `.proto` file.